### ISSUE TITLE
General discussion

### ISSUE THREAD

**Issue by raspstephan** [raspstephan/CBRAIN-Keras-Diagnostics#6]
Hi @gentine, @tbeucler, @mspritch, @mouatadid, @gmooers96,

time for some updates now that I am actively working on the project again. 

I documented the entire workflow from the raw SPCAM training data to the NNCAM simulations here: https://github.com/raspstephan/CBRAIN-CAM/blob/master/notebooks/stephans-devlog/1.0-Entire-workflow-for-32-column-run.ipynb

This should be a great starting point for anyone new to the project. But it also serves to document the status quo and identify workflow bottlenecks and issues in the reproducibility. I already opened some issues (#1, #2, #3, #4) that address some problems in the workflow. 

Talking about issues. I, and anyone working with the code, can add well defined problems that need to be fixed or interesting science enhancements (see e.g. #5). Some of the issues I assigned to myself, but for some I also added the label `help wanted`. These issues would be great projects for collaborators to tackle (i.e. motivated CS students, etc.)

Talking about collaboration, I also started a Wiki for this project: https://github.com/raspstephan/CBRAIN-CAM/wiki
It is pretty sparse so far, but I already added a guide for collaborators: https://github.com/raspstephan/CBRAIN-CAM/wiki/A-guide-for-collaborators

Thanks!

**Comment by raspstephan** [comment]
Now to some science. To go through the workflow, I thought it might be interesting to reproduce the 32 column run crash that I seem to remember. 

As you can see in [this notebook](https://github.com/raspstephan/CBRAIN-CAM/blob/master/notebooks/stephans-devlog/1.0-Entire-workflow-for-32-column-run.ipynb), I first trained a network with only 36 days of data to speed up the computation. When I then ran NNCAM online with this network, the run did NOT crash but it had severe biases (see bottom of the aforementioned notebook). 

Then I trained a network with the entire year of data. See [this notebook](https://github.com/raspstephan/CBRAIN-CAM/blob/master/notebooks/stephans-devlog/1.1-A-closer-look-at-the-32-column-runs.ipynb). The one year-network's performance on the offline validation dataset was quite a bot better than the 36 day version. But the online NNCAM run crashed immediately. See this second notebook for details on the crash. It exhibits the typical crash signal, where a single point will blow up. 

I have no idea why these crashes are so random. I am currently training another one year network with slightly different training settings to check how random the crashes are. This should be a good starting point for stability enhancements such as Noah's loss function.
